CHANGELOG

```diff
@@ -1,4 +1,32 @@
 # Changelog
+## [0.2.5] - 2020-02-19
+### Breaking changes
+ - Deprecate specifying keyword arguments as positional arguments in the public methods:
+   This will become a hard failure in Nmstate-0.3.0 and later.
+   - "libnmstate.apply()"
+   - "libnmstate.commit()"
+   - "libnmstate.rollback()"
+   - "libmstate.show()"
+
+ - NetworkManager 1.22.8 or later is now required.
+
+### New Features
+ - Report missing NM plugin during validation.
+ - Support OVS Link Aggregation (Bonding).
+
+### Bug Fixes
+ - Fix "AtributeError" when editing an existing OVS interface.
+ - Do not remove slaves implicitly when removing the master interface.
+ - Fix team interface port support.
+ - Prevent virtual interface removal after DHCP or IPV6-RA timeout.
+
+### Known Issues
+
+In nmstate 0.2.5 there are some known issues:
```

Comment:
What about the issues from NM side?
- Detaching a system port from an OVS bridge.
- Randomly failing with (connection not found or something).

Reply:
I think the second point was missed.

It is from:
https://bugzilla.redhat.com/show_bug.cgi?id=1804350

The error we saw is `Failed to find a compatible device for this connection`.

Reply:
Done, thanks!